tests/datamodules/test_deepglobelandcover.py

```diff
@@ -0,0 +1,56 @@
+# Copyright (c) Microsoft Corporation. All rights reserved.
```

Comment:
Is this necessary? I've been trying to remove all of `tests/datamodules` since they aren't actually useful tests. Instead, I want to test all datamodules with real trainers in `tests/trainers`.

Reply:
If I check codecov with pytest and remove this datamodule test then the warning we introduced about the batch size is not checked at the moment, otherwise coverage is 100% with only trainer test. So should I add a test for this warning in the trainer and remove this test file?

Reply:
To get coverage of this line, you can either

1. use a batch size that is not a multiple so that the warning is hit, or
2. create two config files (one multiple, one not) and test both.

I would personally prefer the second solution, although the first is faster to test.